### ISSUE TITLE
"Update" section does not correctly verify solution

### ISSUE THREAD

**Issue by SaintPeter** [evanlucas/learnyoumongo#6]
I was excited when my solution to Update worked and I checked my solution against the recommended solution and realized I had an error in my code.  I went back and made changes to my code to make it even more wrong and nothing I did would cause the 'verify' solution to fail.

My bad code which should fail:

``` javascript
var url = 'mongodb://localhost:27017/learnyoumongo';

var mongo = require('mongodb').MongoClient;

mongo.connect(url, function(err, db) {
  if(err) {
    console.log("Error Connecting to DB: ", err);
    db.close();
    return 0;
  }

  var collection = db.collection('docs');

    collection.update({
        bob: 'bob'
    },{
        $set: { age: 41 }
    }, function(err, data) {
        if(err) throw err;
        db.close();    
    });
});
```

**Comment by SaintPeter** [comment]
the 'Remove' section does not correctly verify either.

**Comment by evanlucas** [comment]
Confirmed. I will try and get this fixed as soon as I can.

**Comment by evanlucas** [comment]
Sorry for the delay. Both of those exercises have been fixed in v2.0.0. I did have to make some breaking changes though, so be sure to read the problem carefully :]
